gate-verify/custom-stage-orca/src/main/java/com/opsmx/plugin/stage/custom/VerificationTriggerTask.java

```diff
@@ -180,6 +193,23 @@ private TaskResult triggerAnalysis(StageExecution stage, Map<String, Object> out
 		}
 	}
 
+	private static void verifyStageParams(JsonObject parameters, GateModel gateModel) throws IllegalArgumentException {
+		if (!parameters.has(LOG_TEMPLATE)) {
+			throw new IllegalArgumentException("Verification gate requires a Log Template to be configured");
+		}
+		if (!parameters.has(METRIC_TEMPLATE)) {
+			throw new IllegalArgumentException("Verification gate requires a Metric Template to be configured");
+		}
+		if (!parameters.has(CANARYRESULTSCORE) || parameters.get(CANARYRESULTSCORE).getAsString().isEmpty() || parameters.get(CANARYRESULTSCORE) == null ||
+				!parameters.has(LIFETIME) || parameters.get(LIFETIME).getAsString().isEmpty() || parameters.get(LIFETIME) == null ||
+				!parameters.has(MINICANARYRESULT) || parameters.get(MINICANARYRESULT).getAsString().isEmpty() || parameters.get(MINICANARYRESULT) == null) {
+			throw new IllegalArgumentException("Verification gate in ISD, mandatory fields are missing. Please recheck the stage");
+		}
+		if (!parameters.has(BASELINESTARTTIME) || !parameters.has(CANARYSTARTTIME)) {
```

Comment:
triggerAnalysis() -> getPayloadString() uses start time as baselinestarttime and canarystarttime if they are (or any of them is) missing... with this validation we broke that behavior
@sudhakaropsmx plz review and confirm before @emanipravallika makes changes
cc @aman-agrawal

Reply:
@rakeshkrmeena We don't have much knowledge on Verification gate that time, @emanipravallika verified only one case. 
issue reporter knowledge based @emanipravallika fixed and verified. 
she explained to me then i proceed to merge.
know i asked to @emanipravallika to create new ticket for this fix and same validation we have in sapor service there also we have to fix.